package.json

```diff
@@ -17,14 +18,14 @@
     "mocha:functional": "mocha --timeout 15000 --slow 500 --exit -R tap ./tests/functional_test.js",
     "dev": "npm run watch",
     "watch": "npm-run-all --parallel watch:*",
-    "watch:js": "cross-env NODE_ENV=development nodemon --ext js --ignore tests --ignore public/assets/js/main.dev.js --ignore public/assets/js/bundle.dev.js --exec \"npm run build:development\"",
-    "watch:other": "cross-env NODE_ENV=development nodemon app.js --ext css,pug,svg,png,yml --ignore tests",
-    "prestart": "npm run build:production",
-    "start": "cross-env NODE_ENV=production forever start --plain app.js",
+    "watch:cssjs": "cross-env NODE_ENV=development nodemon --ext css,js --ignore tests --ignore public/assets/css/critical.css --ignore public/assets/js/main.dev.js --ignore public/assets/js/bundle.dev.js --exec \"npm run build:dev\"",
+    "watch:other": "cross-env NODE_ENV=development nodemon app.js --ext pug,svg,png,yml --ignore tests",
+    "start": "npm run build:prod && cross-env NODE_ENV=production forever start --plain app.js",
```

Comment:
I wouldn’t add ‘NODE_ENV=production’ here. That should be picked up from the environment. 

I’d also consider having ‘npm build’ switch on NODE_ENV to run build:prod or build:dev, making start work for both development and production.

Reply:
This was already here (the `NODE_ENV` setup). And in fact we did this from the shell.js era too.

I still need to rethink the scripts part and there's definitely room for improvement. I'll ping you if I don't manage to find a good solution.